spec/services/search/user_spec.rb

```diff
@@ -1,13 +1,13 @@
 require "rails_helper"
 
-RSpec.describe Search::User, type: :service, elasticsearch: true do
+RSpec.describe Search::User, type: :service do
```

Comment:
Minor unrelated optimization. We don't need waste time refreshing Elasticsearch indexes for the first spec so we can move this down to the search doc group of specs.